.buildkite/pipeline.yaml

```
@@ -36,4 +41,22 @@ steps:
       - spec.tar.gz
     plugins:
       - docker#v3.7.0:
-          image: alpine
\ No newline at end of file
+          image: alpine
+
+  - label: ":rocket: Release the spec"
+    # Only execute this step on tags matching "vX.Y"
+    if: build.tag =~ /^v\d+\.\d+$$/
+    command:
+      - *hugo_build_environment_setup
+      # Generate a stable version of the spec
+      - scripts/prepare-spec-release.sh "$BUILDKITE_TAG" spec
+      # Package the build as an artefact, which will be downloaded by:
```

```suggestion
      # Package the build as an artifact, which will be downloaded by:
```

for consistency with `artifact_paths` below